pyproject.toml

```diff
@@ -55,15 +55,15 @@ repository = "https://github.com/conda/conda-build"
 changelog = "https://github.com/conda/conda-build/blob/main/CHANGELOG.md"
 
 [project.scripts]
-conda-build = "conda_build.cli.main_build:main"
-conda-convert = "conda_build.cli.main_convert:main"
-conda-develop = "conda_build.cli.main_develop:main"
-conda-index = "conda_build.cli.main_index:main"
-conda-inspect = "conda_build.cli.main_inspect:main"
-conda-metapackage = "conda_build.cli.main_metapackage:main"
-conda-render = "conda_build.cli.main_render:main"
-conda-skeleton = "conda_build.cli.main_skeleton:main"
-conda-debug = "conda_build.cli.main_debug:main"
+conda-build = "conda_build.cli.main_build:execute"
+conda-convert = "conda_build.cli.main_convert:execute"
+conda-develop = "conda_build.cli.main_develop:execute"
+conda-index = "conda_build.cli.main_index:execute"
```

Comment:
Oh, how do we want to go about `conda-index`?
(The same line is also still in `recipe/meta.yaml` and thus needs adjusting there too.)

The cleanest way would be to
1. add the `conda-index` script to the `conda-index` package,
2. add a `run_constrains` to `conda-index` with `conda-build>=24.1`,
3. release a new version of `conda-index` with those changes,
4. add a that version as the lower bound of the `conda-index` dependency in `conda-build`.

... But IDK, if it's feasible to require a new `conda-index` release before the impending `conda-build=24.1` release....

Reply:
we are already planning on a `conda-index 0.4.0` release to resolve this (and a circular dependency issue for Python 3.12 support)

`conda-build` should no longer be making any `conda-index` entrypoint scripts

Reply:
If it's not possible to do a new `conda-index` release beforehand, we may want to set this as `conda-index = "conda_index.cli.cli"`?

Reply:
as for it still being in `recipe/meta.yaml`, that's a mistake we need to rectify